referrer-policy/no-referrer-policy/no-referrer-policy.subresource.template.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>This page reports back it's request details the parent frame</title>
+    <!-- Common global functions for referrer-policy tests. -->
+    <script src="../generic/common.js"></script>
+  </head>
+  <body style="background:#f4f4f4;">
```

Comment:
1. Why `#f4f4f4`?
2. Why not do this via `<style>body { background ... }</style>`?